stdlib/ballerina-http/src/main/ballerina/ballerina/net/http/resiliency/failover-connector.bal

```diff
@@ -0,0 +1,523 @@
+package ballerina.net.http.resiliency;
+
+import ballerina.net.http;
+
+@Description {value:"Represents a failover connector retry configuration."}
```

Comment:
Shall we say "Represents failover connector retry configuration."

Reply:
Also, we need to be consistent on what we call the connector. Here, it's referred to as `failover connector` and in another place saw that it was referred to as `Failover connector`. Shall we stick to `Failover connector`?